packages/terra-alert/src/Alert.jsx

```diff
@@ -163,7 +163,7 @@ const Alert = ({
         }
       }}
     >
-      <div {...attributes} className={alertClassNames}>
+      <div role="alert" {...attributes} className={alertClassNames}>
```

Comment:
I'm not very knowledgable on the screen reader technology, but didn't @scottwilmarth mention this should be treated with polite to prevent interrupting the user when an alert is rendered and the assistive technology is reading something else?

Reply:
Setting aria-live as polite would function same as the existing behaviour, Once the alert manager is implemented, aria-live  can be set to assertive or polite based on user requirements and severity.